crates/holochain/src/conductor/conductor/builder.rs

```diff
@@ -441,10 +441,11 @@ impl ConductorBuilder {
 
         holochain_p2p.register_handler(handle.clone()).await?;
 
-        // Install extra DNAs, in particular:
-        // the ones with InlineZomes will not be registered in the Wasm DB
-        // and cannot be automatically loaded on conductor restart.
-
+        // Register extra DNAs. In particular, the ones with InlineZomes will
+        // not be registered in the Wasm DB and cannot be automatically loaded
+        // on conductor restart. Hence they need to get passed along here
+        // via the extra_dnas argument (populated from the SweetConductor's
+        // DnaFile cache) in order to be added to the RibosomeStore manually.
```

Comment:
I tried to make that comment clearer because it's not easy otherwise to understand all the custom logic related to inline zomes.